.github/workflows/release-deploy-s3.yml

```diff
@@ -34,8 +34,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: lts
-          cache: yarn
```

Comment:
Any reason for removal of `cache: yarn`? This would/should speed up deployments (if they occur before the cache expires)

Reply:
I don't know how this worked before. 
`cache: yarn` looks for `yarn.lock` in the default directory (which I think is the project root) and throws and error because it is not there. 

The following might work: 
```
cache: yarn
cache-dependency-path: 'deploy/'
```
but are we ever really benefiting from the cache?  This action is ran infrequently

Reply:
The only time there are multiple runs within a cache expiration period is when action errors and we need to test a fix.

Reply:
That's a fair point. Not worth worrying about, was just curious.